File: src/Clients/accountant/src/models/entities/account.ts
import { Syncable } from "models/sync/syncable";

export class Account implements Syncable {
  id: number;
  isMain: boolean;
  stockPrice: number;
  stocks: number;
  balance: number;
  synced = false;

  constructor(public name: string, public currency: string, public createdDate: Date, public modifiedDate: Date) {}
}

File: src/Application/Domain/Accountant/Account.cs
namespace Application.Domain.Accountant;

public class Account : Entity
{
    public int Id { get; set; }
    public int UserId { get; set; }
    public string Name { get; set; }
    public bool IsMain { get; set; }
    public string Currency { get; set; }
    public decimal? StockPrice { get; set; }
}
